tests/pass/shims/x86/intrinsics-x86-bmi.rs

```diff
@@ -0,0 +1,154 @@
+// Ignore everything except x86 and x86_64
+// Any new targets that are added to CI should be ignored here.
+// (We cannot use `cfg`-based tricks here since the `target-feature` flags below only work on x86.)
+//@ignore-target-aarch64
+//@ignore-target-arm
+//@ignore-target-avr
+//@ignore-target-s390x
+//@ignore-target-thumbv7em
+//@ignore-target-wasm32
+//@compile-flags: -C target-feature=+bmi1,+bmi2
+
+#[cfg(target_arch = "x86")]
+use std::arch::x86::*;
+#[cfg(target_arch = "x86_64")]
+use std::arch::x86_64::*;
+
+fn main() {
+    unsafe {
+        test_bmi_32();
+        test_bmi_64();
+    }
+}
+
+unsafe fn test_bmi_32() {
```

Comment:
The tests have zero comments. Is there anything to say about the values being tested or so?

Reply:
Not really. I've added some comments but there is not much worth explaining.